packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1084,17 +1084,17 @@ function detachFiberMutation(fiber: Fiber) {
   // Note that we can't clear child or sibling pointers yet.
   // They're needed for passive effects and for findDOMNode.
   // We defer those fields, and all other cleanup, to the passive phase (see detachFiberAfterEffects).
```

Comment:
This comment block could use an addition for why we don't clear `alternate` here also?